blockchain/chain.go

```diff
@@ -1933,8 +1933,14 @@ func extractDeploymentIDVersions(params *chaincfg.Params) (map[string]uint32, er
 // stxosToScriptSource uses the provided block and spent txo information to
 // create a source of previous transaction scripts and versions spent by the
 // block.
-func stxosToScriptSource(block *dcrutil.Block, stxos []spentTxOut, compressionVersion uint32, isTreasuryEnabled bool) scriptSource {
+func stxosToScriptSource(block *dcrutil.Block, stxos []spentTxOut, compressionVersion uint32, isTreasuryEnabled bool, chainParams *chaincfg.Params) scriptSource {
```

Comment:
Thinking it might be best to only pass `TreasuryVoteInterval` from the params for `stxosToScriptSource` since that's the only field being accessed. Thoughts?

Reply:
I could go either way.

Reply:
I've been tending towards just passing the specific param over the entire chainparams, but it's not a big deal either way.

Reply:
I don't mind either way.  I've already tested this as is though, and it's not a big deal, so unless there is any objection, I'd suggest just leaving it as it is.